File: AirsoftTeamApp/Helpers/RequestResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AirsoftTeamApp.Helpers
{
    public class RequestResult
    {
        public RequestResultEnum Result { get; set; }
        public object Data { get; set; }
        public List<string> Errors { get; set; }
    }
}

File: AirsoftTeamApp/ClientApp/src/app/models/requestResult.ts
import { Component, Inject, Injectable } from '@angular/core';
import { Observable, throwError } from 'rxjs'
import { catchError, tap } from 'rxjs/operators'

export class RequestResult {
  public Data: any;
  public Result: RequestResultEnum;
  public Errors : string[];
}

export enum RequestResultEnum {
  Success = 1,
  Error = 2
}

